feature/floor-map/src/main/java/io/github/droidkaigi/confsched2023/floormap/FloorMapScreenViewModel.kt

```diff
@@ -31,4 +31,8 @@ class FloorMapScreenViewModel @Inject constructor(
             floorLevel = floorLevel,
         )
     }
+
+    fun updateFloorLevelState(floorLevel: FloorLevel) {
```

Comment:
👍

Reply:
This is my preference but how about using onXxxx as ViewModel name for encouraging removing View's logic 👀

Reply:
OK!

Assuming that there are no more than two rooms,
I understood, function with no arguments and change to the another state by executing it.

I have a question, `removing View's logic` also include a recommendation to hold values as properties of the enum in order to reduce the `when` branches using enum within the layout?

Reply:
I mean it is better to be like `onClickFloorLevel(xxxx)`. 
Yes, if we can reduce logic from the Composable function, it would be great.